ddtrace/internal/packages.py

```diff
@@ -10,6 +10,12 @@
 from ddtrace.internal.utils.cache import callonce
 
 
+try:
```

Comment:
🤔 why didn't we do this before?

Is there a high overhead from importing `importlib_metadata`?

Reply:
I don't know, I would love to have feedback on it. I don't think that it can be a problem with the regular setup.
But I was wondering if it could be a serverless issue.

Reply:
Good question, locally it seems to load pretty quick for me, so maybe importing isn't the issue?

Reply:
I'll wait for the gitlab benchmark comments on that PR, it should signal if there is a problem.

Reply:
I'd like to have some review from serverless. It seems there is some regression on that one.
But we will need a fix to still support next Flask version.

Reply:
I changed the solution to use local imports. There is no significant benchmark changes now.